client/zanata-client-commands/src/main/java/org/zanata/client/commands/PutUserOptionsImpl.java

```diff
@@ -33,14 +33,14 @@ public PutUserCommand initCommand() {
     }
 
     @Override
-    @Option(name = "--user-name", required = true,
+    @Option(name = "--user-name", required = false,
```

Comment:
I don't think Java would inherit the parent interface's annotation values, because it doesn't generally inherit annotations at all. 

We should probably remove those redundant Option annotations from the parent interface anyway.